app/src/main/java/com/example/cs4520_twitter/data/Bab.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.cs4520_twitter.data
+
+import java.util.Date
+
+data class Bab(
+    val id: String,
+    val authorUser: User,
+    val content: String,
+    val date: Date,
+    val likes: Int,
+    val likedUserList: List<String>
+)

```

Comment:
Would it be possible to use the `BabEntity` instead in the profile screen along with the other data objects that Thomas defined? That way these dummy data type files can be removed?